### PR TITLE
set n2 survival box sprite layers

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

They looked like regular oxygen boxes before.

## Media

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: N2 survival boxes now show the right tank icon.

### CODE REVIEW

**Review by Errant-4 — CHANGES_REQUESTED** [review]
Please also fix the syndicate one at the end

File: Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml [hunk 1]
```diff
@@ -186,6 +198,11 @@
     - id: Flare
     - id: FoodSnackNutribrick
     - id: DrinkWaterBottleFull
+  - type: Sprite
+    layers:
+    - state: box_hug
+    - state: heart
+    - state: nitrogentank
```
Comment (Errant-4) [inline Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml:205]: Not sure about this one, it looks kinda weird. I think it's okay to just leave the boxhug one with the same look

image

File: Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml [hunk 2]
```diff
@@ -216,6 +228,10 @@
     - id: Flare
     - id: FoodBreadBaguette
     - id: DrinkWaterBottleFull
+  - type: Sprite
+    layers:
+    - state: internals
+    - state: nitrogentank
   - type: Label
     currentLabel: reagent-name-nitrogen
```
Comment (Errant-4) [inline Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml:237]: Also fix BoxSurvivalSyndicateNitrogen in line 289

Comment (IProduceWidgets) [reply Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml:237]: Sure thing.

**Review by Errant-4 — APPROVED** [review]

**Review by VasilisThePikachu — APPROVED** [review]

**Pull request merged**